### DISCUSSION TITLE
Data with 301 features

### DISCUSSION THREAD

**Discussion by HubertPalo** [OxWearables/ssl-wearables#20] [General]
Hello, I've seen you uploaded the data for SSL (in https://zenodo.org/records/6574265 and ssl/ssl_capture_24/data/train/P001.npy) with 301 features, but I failed to comprehend the reason behind. Shouldn't it be just 300?

**Comment by angerhang** [comment]
> Hello, I've seen you uploaded the data for SSL (in https://zenodo.org/records/6574265 and ssl/ssl_capture_24/data/train/P001.npy) with 301 features, but I failed to comprehend the reason behind. Shouldn't it be just 300?

That's correct. The last column is just the standard deviation of each sample. `ssl_capture-24` folder is meant to use capture24 as a pre-training dataset in our original paper, thence the standard deviation was added. If you are only thinking about fine-tuning, feel free to remove the last column.

**Reply by HubertPalo** [reply]
Thank you :)
